plopfile.js

```diff
@@ -61,6 +61,20 @@ module.exports = function(plop) {
           'packages/{{kebabCase name}}/src/test/{{properCase name}}.test.ts',
         templateFile: 'templates/test.hbs.ts',
       },
+      {
+        type: 'modify',
+        path: 'packages/tsconfig.json',
+        transform: (file, {name}) => {
+          const tsConfig = JSON.parse(file);
+          const kebabCase = plop.getHelper('kebabCase');
+          tsConfig.references.push({path: `./${kebabCase(name)}`});
+          tsConfig.references = tsConfig.references.sort(
```

Comment:
`sort` is [in-place](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort), so no need to use the return value.